aidants_connect_habilitation/constants.py

```diff
@@ -0,0 +1,37 @@
+from enum import Enum, unique
+
+
+@unique
+class RequestStatusConstants(Enum):
+    NEW = "Nouvelle"
+    PROCESSING = "En cours"
+    VALIDATED = "Validée"
+    REFUSED = "Refusée"
+    CANCELLED = "Annulée"
+    CHANGES_REQUIRED = "Changements demandés"
+
+    @classmethod
+    def choices(cls):
+        return ((item.name, item.value) for item in cls)
```

Comment:
Tiens je ne connaissais pas `item in cls` non plus

Reply:
Ben j'ai découvert aussi que les enums sont itérables.